core/src/main/java/org/javarosa/core/model/SelectChoice.java

```diff
@@ -121,4 +121,46 @@ public String getTextID() {
     public void setTextID(String textID) {
         this.textID = textID;
     }
+
+    @Override
+    public int hashCode() {
+        int result;
+        result = textID != null ? textID.hashCode() : 0;
+        result = 31 * result + value.hashCode();
```

Comment:
@ctsims Not sure how to determine how much complexity is really necessary here. Some online examples I've found use xor, some don't seem to bother. Any thoughts?